src/subdomains/staking/application/services/staking-withdrawal.service.ts

```diff
@@ -132,11 +135,16 @@ export class StakingWithdrawalService {
     return draftWithdrawals.map((w) => WithdrawalDraftOutputDtoMapper.entityToDto(w));
   }
 
-  async getPendingWithdrawals(): Promise<Withdrawal[]> {
-    return this.withdrawalRepo.getPending();
+  async getStakingWithPendingWithdrawals(): Promise<Staking[]> {
+    return this.stakingRepo
```

Comment:
need to be careful, this will not get full stakings, so you would not be able to preform all operations on staking safely, that's why in other places we were getting stakingIds and then getting staking via repository and not via query.